docs/getting-started/integrating-codacy-with-your-git-workflow.md

```diff
@@ -0,0 +1,61 @@
+---
+description: Configure Codacy to block merging pull requests that don't meet your quality standards.
+---
+
+# Integrating Codacy with your Git workflow
+
+Once you've [customized your setup](customizing-your-setup.md) to best match your needs, you can integrate Codacy with your Git workflow to display analysis results and code coverage on your Git provider and block problematic pull requests.
+
+To integrate Codacy with your Git workflow, follow these steps:
+
+1.  [Configuring quality settings](#configuring-quality-settings)
+1.  [Activating the Git provider integration](#git-provider-integration)
+1.  [Using code coverage as a quality gate](#coverage-gate)
+1.  [Blocking merging pull requests](#blocking-pull-requests)
+
+## 1. Configuring quality settings {: id="configuring-quality-settings"}
+
+[Configure the quality settings](../repositories-configure/adjusting-quality-settings.md) of your project to define the gates (minimum acceptable values) and goals for quality metrics such as issues, duplication, complexity, and code coverage. These gates, goals, and metrics are always visible on Codacy's dashboard and [can be optionally enforced](#blocking-pull-requests) when merging pull requests.
```

Comment:
I'm unsure about adding a link here allowing users to jump directly to the last section of instructions on the page. :thinking: I think it's better to try to encourage a linear sequence.

```suggestion
[Configure the quality settings](../repositories-configure/adjusting-quality-settings.md) of your project to define the gates (minimum acceptable values) and goals for quality metrics such as issues, duplication, complexity, and code coverage. These gates, goals, and metrics are always visible on Codacy's dashboard.
```

Reply:
I was on the fence about the link too, but the fact that they can be optionally enforced is something we decided was useful to point out to the users. I can look into fitting it someplace else.

Reply:
By having a more "in-depth" conceptual discussion in the page intro, the sections can become more lightweight and devoid of details like these.